clash-cores/src/Clash/Cores/Xilinx/DcFifo/BlackBoxes/Internal.hs

```diff
@@ -88,18 +89,22 @@ dcFifoBBF _isD _primName args _resTys = do
   bb :: DcConfig TermLiteralSNat -> BlackBox
   bb dcConfig = BBFunction (show 'dcFifoTF) 0 (dcFifoTF dcConfig)
 
+extrResetPolarity :: N.HWType -> ResetPolarity
```

Comment:
I think I'd roll this up into `DSL.unsafeToHighPolarity` + `DSL.unsafeToLowPolarity`. @DigitalBrains1 disagrees with the names (and I do think he's right) but for consistency's sake let's misname them here too.